### ISSUE TITLE
pikaur and 750 permissions of resulting pkg tarballs

### ISSUE THREAD

**Issue by tblancher** [actionless/pikaur#416]
```
Pikaur v1.5.4
Pacman v5.2.0 - libalpm v12.0.0

```

##### Attached log:

https://git.eldon.me/trey/pastes/src/branch/master/pikaur-S_plex-media-server.output

The problem happens on at least two AUR packages, plex-media-server and unifi.  I've been focusing on plex-media-server, but I think I can demonstrate it for unifi as well if need be.  I also posted this issue to the Arch Linux BBS:  https://bbs.archlinux.org/viewtopic.php?pid=1871117

The problem is with this particular system's pikaur environment.  Running devtools/extra-x86_64-build, or makepkg directly results in the proper 755 permissions in the resulting pkg tarball.  I can manually overcome the situation by issuing two classes of commands:

```sudo chown -R plex:plex /usr/lib/plexmediaserver```
Or
```sudo chmod o=rx -R /usr/lib/plexmediaserver```

I'm sure it's a problem with my pikaur configuration, but I can't figure out what it is.  As stated in the BBS post, my umask for my regular user and root are both 022.  I haven't overridden the `cp` command with a nonstandard alias, and as I said the devtools and makepkg methods generate the pkg tarball with the correct permissions.  What am I missing?  This doesn't happen on my other Arch Linux host where I use pikaur extensively as a drop-in replacement for pacman and installing AUR packages.

UPDATE:  Adding `umask` and `stat` to prepare() and package() of the PKGBUILD (see  BBS post) indicate the umask is OK, but the rpm appears to have the 750 permissions when its unpacked.  I will try and see if I get the same behavior with unifi, the other AUR package I've noticed this in.

**Comment by actionless** [comment]
if the same happens if you build the package manually?

**Comment by actionless** [comment]
also you could use `-v` pikaur flag to see which exactly commands it will run

**Comment by actionless** [comment]
also make sure you're building manually with makepkg in exactly the same directory as pikaur does

to make sure it's not related to mounting options of the filesystems

**Comment by tblancher** [comment]
> also make sure you're building manually with makepkg in exactly the same directory as pikaur does
> 
> to make sure it's not related to mounting options of the filesystems

Good call!  Due to another AUR package (mongodb, which needs nearly 200G free space to build), my ~/.cache/pikaur directory was a symlink to an NFS share with more capacity.  Since I'm no longer building mongodb from source (using mongodb-bin instead), I removed the symlink, and the `stat` command I inserted into the plex-media-server PKGBUILD now shows permissions of 755!  I was getting ready to blame the bsdtar used to extract the foreign package.
